spec/integration/acceptance_spec.rb

```diff
@@ -0,0 +1,76 @@
+require 'spec_helper'
+
+RSpec.describe 'Acceptance test', type: :feature do
+  let(:workdir) { 'tmp/spec' }
+  let(:noid) { '9p2909328' }
+  let(:aip_folder) { "#{workdir}/#{noid}" }
+  let(:aip_file) { "#{aip_folder}.tar" }
+  let(:log_folder) { "#{workdir}/log" }
+
+  let!(:opts) { PushmiPullyu.options.dup }
+  let!(:redis) { Redis.new }
+
+  before do
+    # setup redis and add noid to queue
+    redis.zadd 'test:pmpy_queue', 10, noid
```

Comment:
you might want to clear the queue before the add? It's borderline pointless but I did it in my tests just to ensure if another test missed clearing it after, things would still work as expected.